plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -521,7 +522,8 @@
 
         <xsl:variable name="image-name">
             <xsl:choose>
-                <xsl:when test="lower-case(normalize-space($type)) = ('caution', 'danger', 'note', 'tip', 'warning', 'important')">
+                <xsl:when
```

Comment:
Please rollback all additional linebreaks whch are not related to this ticket (they appeared after you normalised code in the whole file instead of template)